<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppDialog</class>
 <widget class="QDialog" name="AppDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 41, 46);
color: rgb(161, 164, 167);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>951</width>
     <height>851</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="appsLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>196</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
        </property>
        <property name="text">
         <string>Apps</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="appList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 11pt &quot;Noto Sans&quot;;
background-color: #2f535f;
color: rgb(131, 148, 150);
font-weight: 700;
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addApp">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="kres.qrc">
          <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>196</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>196</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
        </property>
        <property name="text">
         <string>Path</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
      <item>
       <widget class="QLabel" name="dataLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>196</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
        </property>
        <property name="text">
         <string>Data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
      <item>
       <widget class="QLabel" name="maskLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>196</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
        </property>
        <property name="text">
         <string>Mask</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="maskText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <item>
       <widget class="QPushButton" name="save">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="kres.qrc">
          <normaloff>:/icons/icons/checkmark.png</normaloff>:/icons/icons/checkmark.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="revert">
        <property name="text">
         <string>Revert</string>
        </property>
        <property name="icon">
         <iconset resource="kres.qrc">
          <normaloff>:/icons/icons/revert.png</normaloff>:/icons/icons/revert.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="kres.qrc">
          <normaloff>:/icons/icons/cancel.png</normaloff>:/icons/icons/cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="kres.qrc"/>
 </resources>
 <connections/>
</ui>
